mkosi.images/base/mkosi.conf.d/10-centos-fedora/mkosi.prepare

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+# SPDX-License-Identifier: LGPL-2.1-or-later
+set -e
+
+if [ ! -d rpm/ ] || [ "$1" = "final" ]; then
+    exit 0
+fi
+
+mkosi-chroot \
+    rpmspec \
+    --with upstream \
+    --query \
+    --buildrequires \
+    --define "_topdir /var/tmp" \
```

Comment:
How does this work in a multi-user system? Is the `_topdir` actually used for anything?
Ah, those are within mkosi chroot, so they are effectively private directories.